prerelease_website/submit_jobs/templates/index.html

```diff
@@ -38,7 +38,13 @@
           Kinetic Kame
         </a>
       </div>
-
+      <div class="col-sm-4 lunar-column text-center">
+        <a href="/kinetic">
```

Comment:
several fixups are in progress on https://github.com/ros-infrastructure/prerelease_website/tree/lunar-fixup, haven't opened the PR yet because it's not ready yet